cli/azd/pkg/commands/pipeline/pipeline_test.go

```diff
@@ -122,8 +155,8 @@ func Test_detectProviders(t *testing.T) {
 		os.Remove(azdoFolder)
 	})
 	t.Run("from persisted data github error", func(t *testing.T) {
-		azdoFolder := path.Join(tempDir, azdoFolder)
-		err := os.Mkdir(azdoFolder, osutil.PermissionDirectory)
+		azdoFolder := filepath.Join(tempDir, azdoFolder)
```

Comment:
Thanks for moving all these file path based operations to use `filepath` and not `path`!